api/resources_portal/test/test_views.py

```diff
@@ -34,6 +34,8 @@ def test_post_request_with_valid_data_succeeds(self):
         eq_(user.username, self.user_data.get("username"))
         ok_(check_password(self.user_data.get("password"), user.password))
 
+        ok_(Organization.objects.get(owner=user))
```

Comment:
Are we using `nose` helpers? Instead of `self.assert...`?

Reply:
Hey man, you wrote the rest of the test!

Reply:
:P

Reply:
hahah yes, just noticed it. It looked weird with the underscore after `ok`

Reply:
But uh, that's a great question. Should we? I tend to follow the existing conventions by default but it appears you didn't set that convention on purpose.

Reply:
yes, this file was also generated by the template, I think I wrote the permission tests with the default methods. I'd lean towards them, but we can update it in a future PR if you want.

Reply:
I switched all the nose asserts over to unittest asserts.